src/api/benchmark/benchmark_test.go

```diff
@@ -0,0 +1,75 @@
+package benchmark
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"github.com/tsenart/vegeta/lib"
+)
+
+func TestGetWalletDirWrk(t *testing.T) {
```

Comment:
you can use the `Benchmark` feature of `go test` for these, see https://dave.cheney.net/2013/06/30/how-to-write-benchmarks-in-go
https://golang.org/pkg/testing/#hdr-Benchmarks